C/plugins/storage/postgres/init.sql

```diff
@@ -879,6 +884,7 @@ INSERT INTO foglamp.statistics ( key, description, value, previous_value )
 insert into foglamp.scheduled_processes ( name, script ) values ( 'COAP', '["services/south"]' );
 -- FogLAMP South Microservice - POLL Plugin template
 insert into foglamp.scheduled_processes ( name, script ) values ( 'POLL', '["services/south"]' );
+insert into foglamp.scheduled_processes ( name, script ) values ( 'CC2650POLL', '["services/south", "--bluetooth_adr", "B0:91:22:EA:79:04"]' );
```

Comment:
Can we put the bluetoth MAC address in configuration rather than as an argument? It would make it much easier for the end user as they have access to the configuration, via the rest api and the ui, but nit to these scheduled_process rows in the database.